pom.xml

```diff
@@ -217,20 +217,18 @@
           </executions>
           <configuration>
             <from>
-              <image>openjdk:11-slim</image>
+              <image>openjdk@sha256:2b6eb2c178908b6704d82258652806eca3c0cd9261df7f5edcc3486d63bc6e60</image>
             </from>
             <to>
               <image>nexus.terrestris.de/shogun/${project.artifactId}:latest</image>
               <tags>
                 <tag>${project.version}</tag>
-                <!--suppress UnresolvedMavenProperty -->
-                <tag>${project.version}-${git.commit.id.abbrev}</tag>
```

Comment:
Any specific reason why you removed this? It was introduced intentionally.

Reply:
Since I've found no proper configuration and a `UnresolvedMavenProperty` was suppressed, I Thought it could be removed.

Or is the config of `spring-boot` uses in SHOGun here?

Reply:
Yes, this element is a default for every submodule (where the plugin is being used, e.g. `shogun-boot`) and the (unresolved) `git` properties will be made available for each one via the git plugin - this is why it's suppressed here.

Reply:
ok